config/environments/production.rb

```diff
@@ -28,8 +28,13 @@
   # config.logger = SyslogLogger.new
 
   # Use a different cache store in production
-  config.cache_store = :mem_cache_store, YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL']
-  config.cache_store = :dalli_store, YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL'] ,
+  config.cache_store = :dalli_store, YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL0'] , 
+                                     YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL1'] , 
+                                     YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL2'] ,
+                                     YAML.load_file("#{Rails.root}/config/local.yml")['MEMCACHED_URL3'] ,
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space found before comma.